dev/pytest/pytest_all_tests.sh

```diff
@@ -0,0 +1,11 @@
+#!/bin/bash
+set -x
+
+# ModelRuntime
+sh dev/pytest/pytest_model_runtime.sh
```

Comment:
How about skipping `sh` and running scripts directly?
Each script is tagged to use `bash` in first line.

```suggestion
dev/pytest/pytest_model_runtime.sh
```

Reply:
I tried it on my Windows, but it seems that bash is not installed by default.
<img width="1312" alt="image" src="https://github.com/langgenius/dify/assets/45712896/e53587e8-814d-420c-a9fc-e55cdfd4374d">

Reply:
any other suggestions? I assume that Windows user have Linux subsystem support on windows. btw, the dev/reformat script is also relies  on bash.

Reply:
Aha, I agree that WSL is quit a good compatibility layer, I like it, but not everyone has installed it.
So, like you mentioned, does the dev/reformat have the same issue?

Reply:
Could you give it a try for running `dev/reformat` on Windows?
And I don't think it would be a blocker issue for this PR anyway.

Reply:
I have skipped the usage of `sh`. You may have a check on latest commits.